OpenEdXMobile/res/drawable/edx_enroll_button.xml

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:state_enabled="false">
+        <shape android:shape="rectangle">
+            <solid android:color="@color/edx_brand_gray_base" />
```

Comment:
I recommend disabled button color, 'back' (#d9d9d9) with text color, 'accent' (#a0a0a0). I spoke with Mark Sadecki, and for disabled buttons, we don't need to follow WCAG rules, which is why we are able to use this coloring.